dapp_manager/cli.py

```diff
@@ -0,0 +1,191 @@
+import sys
```

Comment:
I don't feel any strong benefit from moving the commands to a separate module... especially given that CLI _is_ the only interface that the `dapp-manager` offers... I won't protest though...